simplified-tests/src/test/java/org/nypl/simplified/tests/books/controller/BookRevokeTaskAdobeDRMTest.kt

```diff
@@ -214,9 +216,9 @@ class BookRevokeTaskAdobeDRMTest {
   @Test
   fun testRevokeDRMNonReturnable() {
     val account =
-      Mockito.mock(AccountType::class.java)
+      fakeAccount()
     val accountProvider =
-      Mockito.mock(AccountProviderType::class.java)
+      fakeAccountProvider()
```

Comment:
Maybe we could declare these two as global variables and initialize them on the @before method instead of declaring and initializing them in all the tests.